web/src/styles/scss/section.scss

```diff
@@ -32,6 +32,10 @@ body {
   padding-top: $spacer-2;
   padding-left: $number-padding-size;
   border-top: 1px solid $color-gray-dark;
+
+  &:first-of-type {
+    margin-top: $spacer-2;
+  }
```

Comment:
The first section on the APD builder page (at `/apd`) needs to have less margin from the title to match the mockup.  An attempt was made.  😆